spring-boot-project/spring-boot-docs/src/docs/asciidoc/spring-boot-features.adoc

```diff
@@ -5666,9 +5666,6 @@ If https://hazelcast.com/[Hazelcast] is on the classpath and a suitable configur
 If you define a `com.hazelcast.config.Config` bean, Spring Boot uses that.
 If your configuration defines an instance name, Spring Boot tries to locate an existing instance rather than creating a new one.
 
-If you define a `com.hazelcast.config.Config` bean, Spring Boot uses that.
-If your configuration defines an instance name, Spring Boot tries to locate an existing instance rather than creating a new one.
-
```

Comment:
This shouldn't be removed. If a user specifies an existing `Config` we should use that rather than trying to lookup one.

Reply:
These duplicates.

Reply:
Ah sorry, I missed that. Such polish must be done in a separate commit. I've addressed it in #19678